<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify</class>
 <widget class="QDialog" name="modify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify a Record</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the Customer ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>67</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DL No.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>67</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Car Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Booking Date</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
